### PR TITLE
[FABCN-378] Update links to chaincode JSDoc

### PR DESCRIPTION
**Pull request by jt-nti**
#### Type of change

- Documentation update

#### Description

Update readTheDocs docs chaincode JSDoc links to match new location

https://hyperledger.github.io/fabric-chaincode-node/

#### Related issues

https://jira.hyperledger.org/browse/FABCN-378

### CODE REVIEW

File: docs/source/build_network.rst
Comment (nikhil550) [inline docs/source/build_network.rst:159]: Can you change this link to this release specific url? https://hyperledger.github.io/fabric-chaincode-node/release-1.4/api/

File: docs/source/developapps/smartcontract.md
Comment (nikhil550) [inline docs/source/developapps/smartcontract.md:102]: Same: https://hyperledger.github.io/fabric-chaincode-node/release-1.4/api/

File: docs/source/getting_started.rst
Comment (nikhil550) [inline docs/source/getting_started.rst:29]: Same: https://hyperledger.github.io/fabric-chaincode-node/release-1.4/api/

File: docs/source/tutorial/commercial_paper.md
Comment (nikhil550) [inline docs/source/tutorial/commercial_paper.md:404]: Same

File: docs/source/write_first_app.rst
Comment (nikhil550) [inline docs/source/write_first_app.rst:363]: https://hyperledger.github.io/fabric-chaincode-node/release-1.4/api/

**Review by nikhil550 — APPROVED** [review]

File: docs/source/build_network.rst
Comment (jt-nti) [reply docs/source/build_network.rst:159]: I would rather keep the link to the top level doc for the Node chaincode here rather than straight to a release specific api doc, as for the equivalent Java chaincode link. (Where there were links to specific APIs I did use the release specific urls.)

**Review by nikhil550 — APPROVED** [review]

**Review by pamandrejko — APPROVED** [review]

**Comment by pamandrejko** [comment]
Looks like the mergify bot was not configured for the release-1.4 branch, probably because the Documentation Maintainters PR was not cherry-picked to the release-1.4 branch.
@btl5037 When I went to cherry-pick that PR back to the release-1.4 branch I noticed there was no MAINTAINTERS.md file in that branch.
https://github.com/hyperledger/fabric/tree/release-1.4

So I was not sure what to do?

**Comment by ryjones** [comment]
@pamandrejko I can merge this as an admin while the configuration issues are worked out - do you want me to do that?

**Comment by ryjones** [comment]
@Mergifyio refresh

**Comment by lindluni** [comment]
Correct, we just need to open a PR to update the CODE OWNERS file to match master, we don't need to cherry pick all the maintainers stuff with it. I'll take care of that. Mergify can't merge it because it can't validate against the CODE OWNERS file so branch protection rules haven't passed yet. Ry you can merge if you get to this, it has met the requirements for merge, we just need to fix our codeowners file

**Comment by ryjones** [comment]
Per @btl5037

**Comment by pamandrejko** [comment]
Thanks @ryjones I'll open a PR to update the CODEOWNERS file so that the mergify bot can run against this branch.

**Comment by pamandrejko** [comment]
see PR https://github.com/hyperledger/fabric/pull/620

**Pull request merged**
